src/store/ChartAdapterStore.ts

```diff
@@ -58,15 +163,9 @@ export default class ChartAdapterStore {
             onCrosshairDisappeared: () => {
                 this.mainStore.crosshair.updateVisibility(false);
             },
-            // eslint-disable-next-line @typescript-eslint/no-unused-vars
-            onCrosshairHover: (dx, dy, dxLocal, dyLocal, _indicatorIndex) => {
-                // dxLocal and dyLocal are the local position value correponding to the bottom indicator/main chart
-                const epoch = this.flutterChart?.crosshair.getEpochFromX(dxLocal) || 0;
-                const quote = (this.flutterChart?.crosshair.getQuoteFromY(dyLocal) || 0).toFixed(
-                    this.mainStore.crosshair.decimalPlaces
-                );
-
-                this.mainStore.crosshair.onMouseMove(dx, dy, epoch, quote);
+            onCrosshairHover: (dx, dy, dxLocal, dyLocal, bottomIndicatorIndex) => {
+                const debounced = this.debounce(this.crossHover, 100);
+                debounced(dx, dy, dxLocal, dyLocal, bottomIndicatorIndex);
```

Comment:
can you use debounce from 'lodash.debounce'? We already have that in our package.json

Reply:
Let's not do debounce for tooltip move. Tooltip won't take that much computing and it has to be smooth.
Can we do debounce only for indicator hover check?

Reply:
lodesh.debounce is used now . Debounce for hover is only done now , thanks